src/constants/default-settings.js

```diff
@@ -44,6 +44,7 @@ import {
 
 export const KEPLER_GL_NAME = 'kepler.gl';
 export const KEPLER_GL_VERSION = 'v1.0';
+export const KEPLER_GL_WEBSITE = 'http://kepler.gl/#/';
```

Comment:
Does this need the `/#/` ?

Just curious.

Reply:
I think this is actually needed otherwise it will not kick in react router! Because of github pages